internal/release/promote/interactive_prompt_provider.go

```diff
@@ -182,6 +178,11 @@ func (i *InteractivePromptProvider) ConfirmCreatingPromotionPullRequest() (bool,
 	return selectedAction == actions[0], nil
 }
 
+func (*InteractivePromptProvider) ConfirmAutoMergePullRequest() (answer bool, err error) {
+	err = survey.AskOne(&survey.Confirm{Message: "Do you wish to auto-merge the resulting PR?"}, &answer)
```

Comment:
Do we ever show a diff before creating the PR?

Reply:
Yes the diff is shown just before:
<img width="912" alt="image" src="https://github.com/nestoca/joy/assets/12768642/0997ee08-84e4-4f7f-bd75-c1750add5c86">

Reply:
Ah... yes...